src/test/java/com/endava/internship/mocking/repository/InMemPaymentRepositoryTest.java

```diff
@@ -0,0 +1,73 @@
+package com.endava.internship.mocking.repository;
+
+import com.endava.internship.mocking.model.Payment;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
```

Comment:
Configure Intelij to not use wildcards(for that search 'Code Style' and update 'import count')
![image](https://user-images.githubusercontent.com/114507414/195274395-098343db-fa6a-4201-a505-68ce55d13ed1.png)

Reply:
fixed